tests/system/feature_store/test_feature_store.py

```diff
@@ -244,6 +244,19 @@ def test_serverless_ingest(self):
         stats.remove("timestamp")
         assert features == stats, "didnt infer stats for all features"
 
+    def test_non_partitioned_target_in_dir(self):
+        source = CSVSource(
+            "mycsv", path=os.path.relpath(str(self.assets_path / "testdata.csv"))
+        )
+        path = str(self.results_path / _generate_random_name())
+        target = ParquetTarget(path=path)
+
+        fset = fs.FeatureSet(name="test", entities=[Entity("patient_id")])
+        fs.ingest(fset, source, targets=[target])
+
+        list_files = os.listdir(path)
+        assert len(list_files) == 1
```

Comment:
Can we assert more specifically?

Reply:
not really since the target parquet gets a random name in "path" dir